odoo-shippable/scripts/build-image.sh

```diff
@@ -203,6 +203,10 @@ cat >> /etc/bash.bashrc << EOF
 PYTHONPATH=${PYTHONPATH}:/.repo_requirements/odoo
 EOF
 
+cat >> /etc/zsh/zshrc << EOF
```

Comment:
@moylop260 @JesusZapata could we please use `/root/.zshrc` instead.. 
I've tested, please see the following [asciinema](https://asciinema.org/a/auul5zgm5l0u1lqb9gcntnnqs)